### PR TITLE
Update frontend-core, refactor filter api

### LINKED ISSUE #619: [UI] Access by url is possible when the page is blacklisted
I put a page to the blacklist and as expected I do not see it in side menu but I can see it by url.

### LINKED ISSUE #766: Refactor filter with new Filter API in Frontend-core
We change filter API and we need to use another methods

### PR DESCRIPTION
**Pull request by KlonD90**
Update frontend-core

Refactor page filter API with newer version of API.

In newer version frontend-core: Cache-headers, new page filter API, fix menu bugs like double selection with other modules and no title.

Close #766 
Close #619

### CODE REVIEW

File: webui/src/index.js [hunk 1]
```diff
@@ -56,13 +56,14 @@ class Root extends React.Component {
...
+menuFilter.hideAll()
+
 tarantool_enterprise_core.register(
   PROJECT_NAME,
   menuReducer,
   Root,
   'react',
-  null,
-  menuFilter.check
+  null
```
Comment (aShelist) [inline webui/src/index.js:66]: Do u really need this?

Comment (KlonD90) [reply webui/src/index.js:66]: А в чем вопрос?

File: webui/package.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
     "@graphql-codegen/flow-resolvers": "^1.2.0",
     "@graphql-codegen/introspection": "1.1.3",
     "@graphql-codegen/typescript-compatibility": "^1.2.0",
-    "@tarantool.io/frontend-core": "6.4.0",
+    "@tarantool.io/frontend-core": "^6.5.0",
```
Comment (rgbabaev) [inline webui/package.json:16]: Maybe remove `^`?

File: webui/src/index.js (hunk 1 above)
Comment (rgbabaev) [reply webui/src/index.js:66]: Just to make it a little prettier

**Pull request merged**